### DISCUSSION TITLE
How to include bootstrap manifests from local files?

### DISCUSSION THREAD

**Discussion by vladimirfx** [siderolabs/talos#6770] [Q&A]
Now is two options available to include bootstrap manifests: URLs and inlineManifests (as whole manifest text). 
First one is require host files somewhere, setup header based security witch is difficult and error prone. 
Second is great for most our usecases but makes main config very hard to maintain. We include rendered Cilium config which is many times bigger than Talos own config. In addition content of Cilium manifest available in local (git versioned) file next to Talos config. It would be great if such manifest can be included in Talos config and inlined at apply time.

**Comment by smira** [comment]
good idea, this can be implemented by templating before uploading using one or another option to perform local file templating

**Reply by vladimirfx** [reply]
Somethin simmilar implemented in **talhelper** (for example see https://github.com/budimanjojo/talhelper/issues/68) but it is not work for `inlinedManifests`. Relevant GO Yaml PR: https://github.com/go-yaml/yaml/pull/635
